test/setup/versions.tf

```diff
@@ -19,11 +19,11 @@ terraform {
   required_providers {
     google = {
       source  = "hashicorp/google"
-      version = ">= 4.76.0"
+      version = ">= 6.4"
```

Comment:
@sana-google we typically avoid setting up version in our tests as they are dictated by the modules. You can remove version.tf from example code.

Reply:
I have reverted to how it was before I made the changes. I am not sure how the files were initially set up so I'll leave the decision to delete them up to @yeweidaniel and team.